config/initializers/statsd.rb

```diff
@@ -73,23 +73,18 @@
                      tags: ['version:v0', "context:#{csp}"])
     StatsD.increment(SignIn::Constants::Statsd::STATSD_SIS_CALLBACK_FAILURE, 0,
                      tags: ['version:v0', "context:#{csp}", 'error:'])
-    StatsD.increment(SignIn::Constants::Statsd::STATSD_SIS_TOKEN_SUCCESS, 0,
-                     tags: ['version:v0'])
-    StatsD.increment(SignIn::Constants::Statsd::STATSD_SIS_TOKEN_FAILURE, 0,
-                     tags: ['version:v0', 'error:'])
-    StatsD.increment(SignIn::Constants::Statsd::STATSD_SIS_REFRESH_SUCCESS, 0,
-                     tags: ['version:v0'])
-    StatsD.increment(SignIn::Constants::Statsd::STATSD_SIS_REFRESH_FAILURE, 0,
-                     tags: ['version:v0', 'error:'])
-    StatsD.increment(SignIn::Constants::Statsd::STATSD_SIS_REVOKE_SUCCESS, 0,
-                     tags: ['version:v0'])
-    StatsD.increment(SignIn::Constants::Statsd::STATSD_SIS_REVOKE_FAILURE, 0,
-                     tags: ['version:v0', 'error:'])
-    StatsD.increment(SignIn::Constants::Statsd::STATSD_SIS_INTROSPECT_SUCCESS, 0,
-                     tags: ['version:v0'])
-    StatsD.increment(SignIn::Constants::Statsd::STATSD_SIS_INTROSPECT_FAILURE, 0,
-                     tags: ['version:v0', 'error:'])
   end
+  StatsD.increment(SignIn::Constants::Statsd::STATSD_SIS_AUTHENTICATE_ACCESS_TOKEN_SUCCESS, 0, tags: ['version:v0'])
+  StatsD.increment(SignIn::Constants::Statsd::STATSD_SIS_AUTHENTICATE_ACCESS_TOKEN_FAILURE, 0,
+                   tags: ['version:v0', 'error:'])
```

Comment:
The `error` field in these don't point to anything. Should we just remove this field?